protege-editor-owl/src/main/java/org/protege/editor/owl/model/entity/RandomProlong.java

```diff
@@ -0,0 +1,53 @@
+package org.protege.editor.owl.model.entity;
+
+import org.semanticweb.owlapi.model.OWLEntity;
+
+import uk.org.russet.identitas.*;
+
+public class RandomProlong extends AbstractIDGenerator implements AutoIDGenerator, Revertable {
+
+	
+	private String nextId;
+    public RandomProlong () {
+    	try {
+			nextId = Util.randomProlong();
+		} catch (Exception e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
```

Comment:
Please remove the e.printStackTrace() and user a logger (slf4j) to log exceptions